src/conversions/chrono.rs

```diff
@@ -568,7 +568,7 @@ mod tests {
                         .unwrap()
                         .and_hms_micro_opt(hour, minute, second, ms)
                         .unwrap();
-                    let datetime = DateTime::<Utc>::from_utc(datetime, Utc).to_object(py);
+                    let datetime = Utc.from_utc_datetime(&datetime).to_object(py);
```

Comment:
Is this equivalent to `datetime.and_utc()`? Why use one here and another in the implementation above?

Reply:
Yeah I'm just really tired recently, will make it consistent.